nucypher/characters/chaotic.py

```diff
@@ -132,15 +133,16 @@ class Felix(Character, NucypherTokenActor):
     TEMPLATE_NAME = 'felix.html'
 
     # Intervals
-    DISTRIBUTION_INTERVAL = 60*60   # seconds (60*60=1Hr)
-    DISBURSEMENT_INTERVAL = 24      # (24) hours
+    DISTRIBUTION_INTERVAL = 60  # seconds (60*60=1Hr)
+    DISBURSEMENT_INTERVAL = 24 * 365  # only distribute tokens to the same address once each year.
     STAGING_DELAY = 10              # seconds
 
     # Disbursement
     BATCH_SIZE = 10                 # transactions
-    MULTIPLIER = 0.95               # 5% reduction of previous stake is 0.95, for example
+    MULTIPLIER = Decimal('0.9')     # 10% reduction of previous stake is 0.9, for example
```

Comment:
We can just remove this and the Decimal import now, right?

Reply:
It shouldn't matter but I am aiming to change as little as possible if time is of the essence.